content/rs/security/ldap/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: LDAP authentication
+linkTitle: LDAP authentication
+description: Describes how Redis Software integrates LDAP authentication and authorization.  Also describes how to enable LDAP for your deployment of Redis Software.
+weight: 25
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/administering/designing-production/security/ldap-integration/
+         /rs/security/admin-console-security/ldap/
+         /rs/security/ldap/
+---
+
+As of v6.0.20, Redis Enterprise Software integrates [Lightweight Directory Access Protocol](https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol) (LDAP) authentication and authorization into its role-based access controls (RBAC).  You can now use LDAP to authorize access to the admin console and to authorize database access.
```

Comment:
Link "role-based access controls (RBAC)" to /rs/security/passwords-users-roles/ ?